### DISCUSSION TITLE
Unable to read 'format.json' on Multi-Node Multi-Drive deployment in 2 Linux Mint 20.1 Servers

### DISCUSSION THREAD

**Discussion by kenje4090** [minio/minio#17802] [Q&A] [answered]
Trying to deploy a multinode multidrive using 2 servers with 8 drives each.
Followed [Deploy MinIO: Multi-Node Multi-Drive](https://min.io/docs/minio/linux/operations/install-deploy-manage/deploy-minio-multi-node-multi-drive.html#minio-mnmd) instructions.

- The two servers are on the same LAN. (192.168.3.5 and 192.168.3.6)
- ports 9000 and 9001 on both are open
- no load balancer
- MINIO_SERVER_URL="http://192.168.3.5:9000"
<img width="2543" alt="image" src="https://github.com/minio/minio/assets/59994329/673f3c8e-13c9-4b2c-a104-2e0d5e60f9e9">

## Expected Behavior
Expected behavior is it should start and can access minio dashboard.

## Current Behavior
<img width="2543" alt="Screenshot 2023-08-04 at 2 23 53 PM" src="https://github.com/minio/minio/assets/59994329/1cf465df-2d74-42eb-8815-85f399771333">
- Waiting for a minimum of 8 drives to come online

## Steps to Reproduce (for bugs)
1. Use two Linuxmint 20.1 Servers on the same LAN
2. Follow [Deploy MinIO: Multi-Node Multi-Drive](https://min.io/docs/minio/linux/operations/install-deploy-manage/deploy-minio-multi-node-multi-drive.html#minio-mnmd) instructions.
3. run sudo journalctl -f -u minio.service to see status

## Your Environment
* Version used (`minio --version`): minio version RELEASE.2023-07-21T21-12-44Z (commit-id=d004c4538643eb05ea69b919557d5cf7cc332fae) (for both servers)
* Server setup and configuration: Both servers on LAN
* Operating System and version (`uname -a`): Linux Mint 20.1

**Comment by ravindk89** [comment]
A few things to check:

- Are all drives mounted and readable by the MinIO user?
- Are the drives formatted XFS and empty of any and all files (check for `.<name>` files and folders as well

This looks like both nodes are having drive-level reading issues. Try to resolve those first.

I don't think `MINIO_SERVER_URL` is necessary here, that is more if you set up a reverse proxy or load balancer in front and want to use that hostname

**Reply by kenje4090** [reply]
Hello there, thanks for the response.

Are all drives mounted and readable by the MinIO user? Yes
<img width="1157" alt="image" src="https://github.com/minio/minio/assets/59994329/48f02b31-0c9b-493e-8c1b-3fc7eb195a80">

Are the drives formatted XFS?
- Yes, formatted with mkfs.xfs -f /dev/sda -L DISK1

Are all drives empty of any and all files (check for .<name> files and folders as well
- For server 1, /mnt/disk{1...8} have "minio" folder created after running sudo systemctl start minio.service
- For server 2, /mnt/disk{1...8} have nothing, all empy

**Reply by kenje4090** [reply]
oh and for MINIO_SERVER_URL, I set it based on the comment 

```
# Set to the URL of the load balancer for the MinIO deployment
# This value *must* match across all MinIO servers. If you do
# not have a load balancer, set this value to to any *one* of the
# MinIO hosts in the deployment as a temporary measure.
```

**Reply by kenje4090** [reply]
OHHHH its running now. I forgot to update second server's /etc/default/minio. The first was using /mnt/disk1/minio, the other was just using /mnt/disk1. My bad. Thank you so much

**Comment by kenje4090** [comment]
In Server 192.168.3.5
 - Running telnet 192.168.3.6 9000
    `Connected to 192.168.3.6.`
    
In Server 192.168.3.6
 - Running telnet 192.168.3.5 9000
    `Connected to 192.168.3.5.`

**Comment by kenje4090** [comment, marked as answer]
UPDATE: FIxed. Running now.

Problem was 
- in server 1 /etc/default/minio, it was using http://192.168.3.{5...6}:9000/mnt/disk{1…8}/minio
- in server 2  /etc/default/minio, it was using http://192.168.3.{5...6}:9000/mnt/disk{1…8}/

Solution
- they should be the same
- configured both to use http://192.168.3.{5...6}:9000/mnt/disk{1…8}/minio
- running fine now
